subgraphs/openzeppelin-governor/protocols/angle-governance/src/governor.ts

```diff
@@ -161,3 +164,19 @@ function getGovernanceFramework(contractAddress: string): GovernanceFramework {
 
   return governanceFramework;
 }
+
+function getQuorumFromContract(
+  contractAddress: Address,
+  blockNumber: BigInt
+): BigInt {
+  let contract = Governor.bind(contractAddress);
+  let quorumVotes = contract.quorum(blockNumber);
+
+  let governanceFramework = getGovernanceFramework(
+    contractAddress.toHexString()
+  );
+  governanceFramework.quorumVotes = quorumVotes;
```

Comment:
Minor refactor to update the `governanceFramework` level `quorumVotes` so its updated out of date